src/app/controllers/users/create.js

```diff
@@ -1,5 +1,29 @@
 'use strict';
 
+flnd.userCreate = {
+  configureSubmit: function(config, $scope, messenger, authService, $uibModalInstance) {
+    return function() {
+        $scope.loading = true;
+
+        authService.post(config.users, $scope.user)
+            .then(function success() {
+
+                $uibModalInstance.close();
```

Comment:
Previously I was creating a save message in the page when save succeeded. Now that I've started converting the edit forms into modal dialogs, I've moved the "success" message to the main list page, and I just close the modal popup.